### PR TITLE
fix issue 650 for develop

### PR DESCRIPTION
**Pull request by asroy**
Fixing issue #650. 

Beware of the culprit
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/650#issuecomment-754979896
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/650#issuecomment-756987392

### CODE REVIEW

**Review by jeffdaily — DISMISSED** [review]
LGTM

File: src/kernels/composable_kernel/include/utility/amd_buffer_addressing.hpp [hunk 1]
```diff
@@ -139,12 +139,7 @@ __llvm_amdgcn_buffer_store_bf16x4(ushort4_t vdata,
                                   bool slc) __asm("llvm.amdgcn.buffer.store.v4bf16");
 
 #if CK_USE_AMD_BUFFER_ATOMIC_FADD
-#if CK_HIP_VERSION_FLAT >= 3010020405
```
Comment (atamazov) [inline src/kernels/composable_kernel/include/utility/amd_buffer_addressing.hpp:142]: Please look at question at https://github.com/ROCmSoftwarePlatform/MIOpen/issues/650#issuecomment-757013445. Perhaps we need to use the current version from the staging.

Comment (asroy) [reply src/kernels/composable_kernel/include/utility/amd_buffer_addressing.hpp:142]: fixed

**Review by atamazov — APPROVED** [review]
LGTM

**Comment by daniellowell** [comment]
This PR has tidy issues

**Comment by asroy** [comment]
@daniellowell CI passed

**Pull request merged**